modules/admin_manual/pages/_partials/configuration/server/occ_command/system_command.adoc

```diff
@@ -0,0 +1,22 @@
+= System
+
+To execute xref:configuration/server/background_jobs_configuration.adoc[background jobs] using xref:configuration/server/background_jobs_configuration.adoc#cron[cron], you can use the `system:cron` command, as in the following example:
+
+[source,console]
+....
+sudo -u www-data php occ system:cron
+....
+
+[IMPORTANT]
+====
+The `system:cron` command replaces https://doc.owncloud.com/server/admin_manual/configuration/server/background_jobs_configuration.html#cron[System cron], which used `</path/to/your/owncloud>/cron.php`.
```

Comment:
I guess this impacts how the command is used in crontab - or?
In case, pls show how was the command was used before and now.

Reply:
It doesn't seem necessary to show that information here, as it's available in the linked section of the docs.

Reply:
it is nicly described in Server 10.3 Release Notes https://github.com/owncloud/docs/pull/1881 at about line 62,
so why not here where it makes a lot of sense?
btw the linked section tells using cron.php...

Reply:
Your advice sounds good to me. I've made a further change that integrates the content that you referred to. Right to merge?